packages/gatsby-source-wordpress/src/models/remoteSchema.ts

```diff
@@ -97,6 +98,7 @@ const remoteSchema: IRemoteSchemaStore = {
       internal: `wpInternal`,
       plugin: `wpPlugin`,
       actionOptions: `wpActionOptions`,
+      fields: 'wpFields',
```

Comment:
@tinytoolbox This fails our lint checks, please change:
```suggestion
      fields: `wpFields`,
```

Reply:
fix applied.  Thanks